### ISSUE TITLE
"table" class mangles signpost styling

### ISSUE THREAD

**Issue by ezekeal** [vmware/clarity#4698]
## Describe the bug

When a signpost is used:
- inside of a table header
- and the table has the class "table"

The styling of the signpost is mangled

## How to reproduce

[stackblitz reproduction](https://stackblitz.com/edit/clarity-v3-dark-theme-rwndwf)

1.  create a table
2.  set the table's class to "table"
3.  create a table header
4.  add a signpost to the head
5.  click signpost to see mangled formatting

## Expected behavior

signpost will retain formatting regardless of outer element

## Versions

**App**

- Angular: 9
- Node: 12.18.0
- Clarity: 3

**Device:**

- Type: mbp
- OS: Mac OS 10.15.4
- Browser chrome
- Version 81

## Additional notes

none

**Comment by gnomeontherun** [comment]
Yes, some styles are set on the table header cells that bleed into the signpost content areas. This should be something you can workaround in the CSS but it is likely to be fixed by #1757.
